src/patternfly/components/ClipboardCopy/examples/ClipboardCopy.md

```diff
@@ -10,17 +10,19 @@ cssPrefix: pf-c-clipboard-copy
   {{#> clipboard-copy-group}}
```

Comment:
Can you make the example ids reference a descriptive name rather than a number? For the examples, instead of "1, 2, 3..." can you use the `title` + `-example` for the `id` which would be `clipboard-copy--id="collapsed-example"`? Then update the examples to use `clipboard-copy--id` as the first value to `concat`? That way, the code is much easier to maintain in that you don't have to worry about indexing, just change the root and reuse `id` concat statement. So if you added an example between 2 and 3, you don't have to update 3, 4, 5 etc. It's also easier to debug when reading ids the reference the example you're looking at.

ids would read 
- `id="collapsed-example-copy-button"`
- `id="collapsed-example-text-input"`
etc

As opposed to 
- `id="copy-button-4"`
- `id="text-input-4"`
etc